### PR TITLE
Add Exercise01

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Exercise01/Program.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise01
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Func<int, List<int>> ListOfPrimeSmallerX = (x) =>
```
Comment (voquanghoa) [inline Exercise01/Program.cs]: Make `ListOfPrimeSmallerX` to become a class function

File: Exercise01/Program.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+            var newList = ListOfPrimeSmallerX(100);
+
+            newList.ForEach(Console.WriteLine);
+
+            Console.ReadKey();
+        }
+    }
+
+    public static class Extention
```
Comment (voquanghoa) [inline Exercise01/Program.cs]: Move this class to a new file

File: Exercise01/Program.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply Exercise01/Program.cs]: Done

File: Exercise01/Program.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply Exercise01/Program.cs]: Done

File: Exercise01/Program.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            ListOfPrimeSmallerX listOfPrimeSmallerX = (x) =>
+            {
+                var list = new List<int>();
+
+                for (int i = 2; i <= x - 1; i++)
```
Comment (voquanghoa) [inline Exercise01/Program.cs]: Replace `i<= x -1` by `i<x`

File: Exercise01/Program.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+namespace Exercise01
+{
+    public delegate List<int> ListOfPrimeSmallerX(int i);
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            ListOfPrimeSmallerX listOfPrimeSmallerX = (x) =>
```
Comment (voquanghoa) [inline Exercise01/Program.cs]: Make a function `private static List<int> GetAllPrimeNumbersSmallerThan(int x)`

Comment (huynhngocminh2511) [reply Exercise01/Program.cs]: Done

File: Exercise01/Program.cs (hunk 3 above)
Comment (huynhngocminh2511) [reply Exercise01/Program.cs]: Done

**Pull request merged**